src/client.ts

```diff
@@ -343,8 +347,8 @@ class OptimizelyReactSDKClient implements ReactSDKClient {
       }
 
       if (userInfo.id) {
-        userContext = this._client.createUserContext(userInfo.id, userInfo.attributes);
-        return userContext;
+        this.userContext = this._client.createUserContext(userInfo.id, userInfo.attributes);
```

Comment:
Do we want to replace existing userContext if non-matching userId is requested?
It can be a breaking change.

Reply:
yeah, I have the same question as Jae

Reply:
Correction made.